script.js

```diff
@@ -50,23 +50,52 @@ document.addEventListener('DOMContentLoaded', () => {
   /** ****    Form Validation       ***** */
   const form = document.querySelector('#form');
   const formSubmit = form.querySelector('.form-submit');
+  const name = form.querySelector('#name');
   const email = form.querySelector('#email');
+  const message = form.querySelector('#message');
   const showError = form.querySelector('span');
+
   function emailLowerCase() {
     const emailValue = email.value;
     if (emailValue.toLowerCase() === emailValue) {
       showError.innerText = '';
       return true;
     }
     showError.classList.add('error');
-    showError.innerText = `Email should be in lowercase. You need to type like this ${
-      emailValue.toLowerCase()}`;
+    showError.innerText = `Email should be in lowercase. You need to type like this ${emailValue.toLowerCase()}`;
     return false;
   }
+
   formSubmit.addEventListener('click', (event) => {
     event.preventDefault();
```

Comment:
- [ ] [Optional] Congratulations on successfully adding local storage functionality to your form inputs! I did notice that the validation rules specified in the HTML file are not being applied as expected because the form submission's default behavior is being prevented. Please ensure that these validation rules are applied to your form input files, just like you did for the email lowercase validation. 🙏